src/RPackage-Core/RPackageOrganizer.class.st

```diff
@@ -213,6 +214,15 @@ RPackageOrganizer >> addMethod: method [
 	rPackage addMethod: method
 ]
 
+{ #category : #'package-names-cache' }
+RPackageOrganizer >> addPackageNameToCache: aPackageName [
+
+	packageNames ifNil: [ ^ self ].
```

Comment:
why not 
```Smalltalk
self packageNames add: ... 
```
?

Reply:
Because packageNames will force the generation of the cache during the loading of code, and I wanted to generate it in the first read. Maybe I am just optimizing before needed.